### PR TITLE
pretty water and various fixes

### PR DESCRIPTION
**Pull request by buffyuwu**
## About The Pull Request
adds a depth system so that we might eventually have varying levels/depths of water
adds animated water effects when entering water
fixes arcade sounds to no longer play at 100 volume because dear god why was this a thing
also fixes some annoying bos bunker things like a bench under a chair and a floating camera
adds an autodrobe to the BoS bunker

## Pre-Merge Checklist
- [x] Your Pull Request contains no breaking changes
- [x] You tested your changes locally, and they work.
- [x] There are no new Runtimes that appear.
- [x] You documented all of your changes.

## Changelog
🆑 
add: adds animated water effects when entering water
tweak: arcade sounds no longer play at 100 volume because dear god why was this a thing
/:cl:

https://user-images.githubusercontent.com/42501819/135533907-5873c72d-0b96-4d09-9f33-ad016af43305.mp4

### CODE REVIEW

**Review by hecksadecimal — CHANGES_REQUESTED** [review]
Use the PR template.

File: code/modules/vending/autodrobe.dm [hunk 1]
```diff
@@ -17,6 +17,7 @@
 						/obj/item/clothing/under/costume/schoolgirl/red = 1,
 						/obj/item/clothing/under/costume/schoolgirl/green = 1,
 						/obj/item/clothing/under/costume/schoolgirl/orange = 1,
+						/obj/item/clothing/head/kitty = 1,
```
Comment (ThyBaron) [inline code/modules/vending/autodrobe.dm:20]: No.

**Review by ThyBaron — CHANGES_REQUESTED** [review]
Please remove the availability of kitten ears! For reasons I won't explain in great detail, implementing kitten ears will give the wrong effect, and is not helpful for our atmospheric progress thus far.

**Review by Looreydev — CHANGES_REQUESTED** [review]
Use the full Pull Request Template as it is. This is simply a Standard that should be enforced.

**Review by hecksadecimal — APPROVED** [review]

**Pull request merged**